packages/account/tests/account.e2e.spec.ts

```diff
@@ -90,20 +86,73 @@ describe("Account Tests", () => {
     };
 
     const balance = (await checkBalance(publicClient, recipient, nftAddress)) as bigint;
-    const partialUserOp = await smartWallet.buildUserOp([transaction]);
 
-    const paymasterData = await paymaster.getPaymasterAndData(partialUserOp, {
-      mode: PaymasterMode.SPONSORED,
+    const maticBalanceBefore = publicClient.getBalance({ address: await smartWallet.getAddress() });
+
+    const response = await smartWallet.sendTransaction(transaction, { paymasterServiceData: { mode: PaymasterMode.SPONSORED } });
+
+    const userOpReceipt = await response.wait(3);
```

Comment:
Just out of curiosity, why are we waiting for 3 blocks here?

Reply:
No specific reason, just wanted to make sure the oser op is sent before proceeding, i can change it.